gson/src/main/java/com/google/gson/internal/bind/ReflectiveTypeAdapterFactory.java

```diff
@@ -394,6 +391,8 @@ T finalize(T accumulator) {
   }
 
   private static final class RecordAdapter<T> extends Adapter<T, Object[]> {
+    static Map<Class<?>, Object> ZEROES = primitiveZeroes();
```

Comment:
Maybe something like `PRIMITIVE_DEFAULTS` or similar would be a bit more descriptive but it does not matter much.

Reply:
OK, done.